### ISSUE TITLE
Services societaux

### ISSUE THREAD

**Issue by nosgestesclimat** [datagir/nosgestesclimat#1625]
Conso pour 1 individu 1,4 Tpour conso préconisée 2T
> Depuis la page: `https://nosgestesclimat.fr/documentation/bilan`
> Ce ticket a été créé automatiquement par notre robot depuis notre [page de contribution](https://nosgestesclimat.fr/contribuer).

**Comment by EmileRolley** [comment]
L'objectif de 2 tonnes CO2e est fixé pour horizon 2050, or les 1.4 tonnes CO2e des services sociétaux correspond à l'état actuel de la France. Pour éteindre l'objectif il faudra une métamorphose du système actuel, et donc, une nette diminution de la part des services sociétaux dans l'empreinte de la France.

**Comment by Clemog** [comment]
A mon avis, il est important de modifier la [FAQ](https://nosgestesclimat.fr/contribuer#:~:text=Pourquoi%20ne%20pas%20prendre%20en%20compte%20l%27%C3%A9pargne%20individuelle%20%3F) pour prendre en compte cette nouvelle valeur des services sociétaux (anciennement services publics)

**Comment by laem** [comment]
@Clemog  On a déjà une question dans la FAQ à ce sujet. 

![image](https://user-images.githubusercontent.com/1177762/210263499-54f8630a-4bc5-4fd0-8bbb-b230f39a0cad.png)


Tu penses qu'ajouter le chiffre (qui va changer...) résoud l'incompréhension ?

**Comment by laem** [comment]
À ce sujet, il faudrait qu'on intègre dans l'écran de fin le sujet de l'aire sous la courbe. 
C'est le seul objectif qui vaille en fait, loin de cet horizon des 2 tonnes qui n'est que le premier niveau de lecture, et qui est potentiellement faux et trompeur. 

Voir cet excellent fil imagé https://twitter.com/AEffondrement/status/1608751812097748998

En bref, reprendre le message de notre article de blog dans l'écran de fin. 

https://datagir.ademe.fr/blog/budget-empreinte-carbone-c-est-quoi/

**Comment by Clemog** [comment]
> Tu penses qu'ajouter le chiffre (qui va changer...) résoud l'incompréhension ?

Je pense qu'on pourrait être plus précis, rediriger vers la documentation par exemple qui permet d'aller plus loin pour vraiment comprendre davantage ce que poste contient réellement 

Et on pourrait aussi rediriger dans la FAQ vers l'empreinte carbone nationale top down ou des pages de docs en général

**Comment by Clemog** [comment]
Ça revient à une autre issue sur l'amélioration de la FAQ je pense mais je trouve que c'est problématique de ne pas pouvoir linker facilement une question précise de la FAQ (on est obligé de prendre un screen ou partager la page entière ce que je trouvais problématique lors de l'envoi de mails) !

**Comment by laem** [comment]
> de ne pas pouvoir linker facilement une question précise de la FAQ

Oui, en fait on a des `ids` sur les questions, mais étant donné qu'on va chercher la FAQ au runtime, les liens du type https://nosgestesclimat.fr/contribuer#bio ne marchent qu'une fois la page chargée... donc pas très utile.
